Lib/test/test_xml_etree.py

```diff
@@ -536,6 +536,7 @@ def test_iterparse(self):
         iterparse = ET.iterparse
 
         context = iterparse(SIMPLE_XMLFILE)
+        self.assertIsNone(context.root)
         action, elem = next(context)
```

Comment:
It is still None until the end.
```suggestion
        action, elem = next(context)
        self.assertIsNone(context.root)
```